### PR TITLE
Add Orlando file

### LINKED ISSUE #6: YOUR_USERNAME tracks progress on GH Flow
- [x] create a **feature branch**
- [x] add some **commits**
- [x] open a pull request
- [x] collaborate and add additional commits
- [x] merge my pull request with 1 👍

### PR DESCRIPTION
**Pull request by githubteacher**
This PR adds the Orlando file to our project. This PR closes #6 

Some ❓s:
1. If you've been to Orlando before, what did you like? what did you not like?
2. If you have children, what might be of interest to you? What do you want to know before you visit Orlando?
3. @hollenberry do you know if there's any attractions unique to FL that you don't have in CA?

### CODE REVIEW

File: YOUR_USERNAME-orlando.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Orlando, FL
+
+### Restaurants
+
+- The Pharmacy is a super hidden spot
+- Rocco's Tacos
+
+### Local Attractions
+
+- Gatorland :crocodile:
```
Comment (alanzhong) [inline YOUR_USERNAME-orlando.md:14]: :O

**Comment by mmanfred** [comment]
👍 
kids want a pool and roller coasters

**Comment by githubteacher** [comment]
@mmanfred take a look at 31b3491

**Comment by hollenberry** [comment]
> do you know if there's any attractions unique to FL that you don't have in CA?

We have two Zoos in San Diego, and one has Pandas. Do you have that in FL?

This looks good to 🚢 otherwise! 👍

**Comment by mmanfred** [comment]
looks good to me 👍 🚢 :shipit:  🚀 😲 🍔 ⚡

**Comment by githubteacher** [comment]
> We have two Zoos in San Diego, and one has Pandas. Do you have that in FL?

@hollenberry I'll add this in the next iteration.

**Pull request merged**